packages/clarity-js/src/data/variable.ts

```diff
@@ -44,3 +63,22 @@ export function reset(): void {
 export function stop(): void {
     reset();
 }
+
+function redact(input: string): string {
+    return input.length >= Setting.WordLength ? 
+        `${input.substring(0,2)}${scrub(input.substring(2), Constant.Asterix, Constant.Asterix)}` : scrub(input, Constant.Asterix, Constant.Asterix);
+}
+
+async function sha256(input: string): Promise<string> {
+    try {
+        if (crypto) {
+            // Reference: https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest#converting_a_digest_to_a_hex_string
+            const buffer = await crypto.subtle.digest(Constant.SHA256, new (TextEncoder as any)(Constant.UTF8).encode(input));
```

Comment:
The default for TextEncoder is UTF8 and I think they removed support for everything else. So, you can call it without passing UTF8 and avoid the type casting to any.

Reply:
Good point, fixed.